### ISSUE TITLE
Migrate discuss.kubernetes.io to Google Analytics 4 (GA4)

### ISSUE THREAD

**Issue by chalin** [kubernetes/community#6965]
### Describe the issue

This issue is part of a CNCF-wide effort to upgrade project websites to GA4 since **Google has deprecated Universal Analytics** (UA). For more details, see:

- https://github.com/cncf/techdocs/issues/108, including reference to a suggested [migration process](https://github.com/cncf/techdocs/blob/main/docs/analytics.md#migrating-to-google-analytics-4), which we've used to [migrate a few CNCF projects](https://docs.google.com/spreadsheets/d/1Mx4LhdI2Un-rvGMI73SlHxQH9D2HABAJclMB3dd6lnA) already.

### Tasks

- [ ] [Connect] the UA ID `UA-98841695-3` from the [GA4 web data stream `G-JPP6RFM2BP`][ga4-web]
- [ ] (Optional) From the [UA console], filter out all events not from the discuss server
- [ ] Configure the Discourse server to use the GA4 ID `G-JPP6RFM2BP`
- [ ] Confirm that the GA4 data stream is receiving events from the discuss server
- [ ] Confirm that the UA site tag is still receiving events

### Notes

- Corresponding issue for main site: https://github.com/kubernetes/website/issues/35299
- Followup: Drop dis[connect] the UA ID once it is no longer needed
- I'm not sure that this is the right repo to create this issue in, but it seemed like the most reasonable choice. If the issue needs to be moved, please let me know.

### Pre-migration website analytics info

- UA ID: `UA-98841695-3` ([UA console])

/cc @caniszczyk @nate-double-u

[connect]: https://support.google.com/analytics/answer/9973999
[Netlify snippet injection]: https://docs.netlify.com/site-deploys/post-processing/snippet-injection/
[ga4-web]: https://analytics.google.com/analytics/web/#/p327084191
[UA console]: https://analytics.google.com/analytics/web/#/report-home/a98841695w183024907p180630334

**Comment by chalin** [comment]
@jimangel @jberkus @a-mccarthy or anyone else - can you perform the "Configure Discourse server to use the GA4 ID" task listed in the opening comment and confirm once it is done?

**Comment by jberkus** [comment]
@chalin this will take a while to figure out, because I didn't originally set up the GA integration, and apparently Discourse has no current documentation on how to do this.

**Comment by mrbobbytables** [comment]
I've removed the GA tracking code altogether. I think it was pointing towards a personal one for testing ages and ages ago but never got moved over. We've never looked at the GA for it, just its built in metrics^^;;;

**Comment by jberkus** [comment]
Hah, that was the new code.

Are you saying we don't want GA for it?  Fine with me.

**Comment by chalin** [comment]
The old ID was `UA-98841695-3`. Of course, if you don't want analytics, that's fine. If you do want analytics, then @mrbobbytables, could you use the new ID `G-JPP6RFM2BP` where ever you removed the old ID from (assuming the gtag.js library is being used)?

**Comment by chalin** [comment]
So @mrbobbytables and @jberkus - is the official position that [discuss.kubernetes.io](https://discuss.kubernetes.io/) will have no analytics? If so, that's fine, I'll just close this issue in that case.

**Comment by mrbobbytables** [comment]
Yup! I removed it already, we weren't really using the analytics to begin with (didn't even know it was enabled).
